### ISSUE TITLE
handle broken images

### ISSUE THREAD

**Issue by luwen-huang** [wombats-writing-code/fbw-student-app#3]
if image can't be found, store needs to throw away that one and refetch a new Next item.

super annoying: 
![screen shot 2016-07-23 at 12 23 21 pm](https://cloud.githubusercontent.com/assets/2822290/17074558/7fdcb65c-50d0-11e6-985a-d94f8621ae72.png)

**Comment by cjshawMIT** [comment]
There's a bigger issue behind this -- the image should be loading. While it might be good UX to always present a valid question, unless we solve the root cause, that may just mean we never show a question with an image -- unacceptable to the faculty, I'm sure.

**Comment by cjshawMIT** [comment]
I think the issue should be fix-root-cause-of-why-images-don't-show-up, not ignore-broken-images

**Comment by luwen-huang** [comment]
i thought the image isn't loading because something got garbled in the image processing pipeline, e.g. corrupted image, cdn couldn't find path, etc. something that's out of our control? because other images are loading.

**Comment by cjshawMIT** [comment]
Hmmm…..I thought it was happening to all images. How would we know that the image isn’t available for a given problem, since that’s an error that would show up in the WebView only? Let me investigate more on Monday ...

**Comment by luwen-huang** [comment]
nope, it's for that specific image only. the question before it (what is the y-intercept of the graph shown below) shows up fine. actually, i think fetch throws an error and says the image cannot be found, so we can handle it. take a look. (note that i've passed the error handling in fbw-utils up one level, so you might need to log it there)

**Comment by cjshawMIT** [comment]
That image shows up fine in the direct [qbank call](https://qbank-dev.mit.edu/touchstone/api/v2/repository/repositories/repository.Repository:57279fb9e7dde086d01b93ef@bazzim.MIT.EDU/assets/repository.Asset:577c26f671e4822100d6b52c@bazzim.MIT.EDU/url), so I wonder if it has something to do with the fact that it's in a choice?

**Comment by cjshawMIT** [comment]
The issue seems to be that in the process of the question getting to the app, the choices all get munged into a single text blob ... something isn't escaped properly. The good news is that the item appears fine in the database.

```
choices: 
      [ { text: '<p><img alt="" src="https://qbank-dev.mit.edu/api/v2/repository/repositories/repository.Repository:57279fb9e7dde086d01b93ef@bazzim.MIT.EDU/assets/repository.Asset:577c274171e4822100d6b530@bazzim.MIT.EDU/url%5C%22%20style=%5C%22height:255px;%20width:350px%5C%22%20/%3E%3C/p%3E%5Cn%22,%22id%22:%22577c279471e4822100d6b545%22,%22name%22:%22Choice%202%22%7D,%7B%22text%22:%22%3Cp%3E%3Cimg%20alt=%5C%22%5C%22%20src=%5C%22https://mc3.mit.edu/fbw-author/api/v2/repository/repositories/repository.Repository:57279fb9e7dde086d01b93ef@bazzim.MIT.EDU/assets/repository.Asset:577c26f671e4822100d6b52c@bazzim.MIT.EDU/url%5C%22%20style=%5C%22height:255px;%20width:350px%5C%22%20/%3E%E2%80%8B%E2%80%8B%E2%80%8B%E2%80%8B%E2%80%8B%E2%80%8B%E2%80%8B%3C/p%3E%5Cn%22,%22id%22:%22577c279471e4822100d6b543%22,%22name%22:%22Choice%200%22%7D,%7B%22text%22:%22%3Cp%3E%3Cimg%20alt=%5C%22%5C%22%20src=%5C%22https://mc3.mit.edu/fbw-author/api/v2/repository/repositories/repository.Repository:57279fb9e7dde086d01b93ef@bazzim.MIT.EDU/assets/repository.Asset:577c272071e4822100d6b52e@bazzim.MIT.EDU/url%5C%22%20style=%5C%22height:255px;%20width:350px%5C%22%20/%3E%3C/p%3E%5Cn%22,%22id%22:%22577c279471e4822100d6b544%22,%22name%22:%22Choice%201%22%7D,%7B%22text%22:%22%3Cp%3E%3Cimg%20alt=%5C%22%5C%22%20src=%5C%22https://mc3.mit.edu/fbw-author/api/v2/repository/repositories/repository.Repository:57279fb9e7dde086d01b93ef@bazzim.MIT.EDU/assets/repository.Asset:577c278e71e4822100d6b532@bazzim.MIT.EDU/url" style="height:255px; width:350px" /></p>\n',
          id: '577c279471e4822100d6b546',
          name: 'Choice 3' } ],
```

**Comment by cjshawMIT** [comment]
fixed in commit 7122656dda03dddf92dba8cf57ec690c19a43687
